src/dashboard/Data/Browser/BrowserToolbar.react.js

```diff
@@ -138,6 +139,16 @@ let BrowserToolbar = ({
           onClick={onCloneSelectedRows}
         />
         <Separator />
+        <MenuItem
+          disabled={!selectionLength}
+          text={`Export ${selectionLength <= 1 ? 'this row' : 'these rows'} to CSV`}
```

Comment:
I think this great feature merits its own menu item! Instead of placing it under `Edit`, how about creating a new `Export` item and moving the two sub-items there?

I would add it between `Browse` and `Refresh` and we should have an existing icon with an arrow pointing downwards.

Reply:
Got it, will post this in the main conversation area for review.